integrations/docker/images/chip-build-ameba/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN set -x \
     && cd ${AMEBA_DIR} \
     && git clone --progress -b cmake_build https://github.com/pankore/ambd_sdk_with_chip_non_NDA.git \
     && cd ambd_sdk_with_chip_non_NDA \
-    && git reset --hard a57f9da \
```

Comment:
consider using a tag (not a branch since that makes things unstable) for more clarity as things stabilise.

Reply:
I have modified this using a tag, thanks!